Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputControllerOSX.cs

```diff
@@ -0,0 +1,115 @@
+#if UNITY_STANDALONE_OSX || UNITY_EDITOR_OSX
+using System.Runtime.InteropServices;
+using UnityEngine.Experimental.Input.Layouts;
+using UnityEngine.Experimental.Input.Plugins.XInput.LowLevel;
+using UnityEngine.Experimental.Input.Utilities;
+
+
+namespace UnityEngine.Experimental.Input.Plugins.XInput.LowLevel
+{
+    // Xbox one controller on OSX. State layout can be found here:
+    // https://github.com/360Controller/360Controller/blob/master/360Controller/ControlStruct.h
+    // struct InputReport
+    // {
+    //     byte command;
+    //     byte size;
+    //     short buttons;
+    //     byte triggerLeft;
+    //     byte triggerRight;
+    //     short leftX;
+    //     short leftY;
+    //     short rightX;
+    //     short rightY;
+    // }
+    // Report size is 14 bytes. First two bytes are header information for the report.
```

Comment:
Use XML doc comments. Turn link into actual link and code into `<code>` snippet.